src/authenticate.cpp

```diff
@@ -54,7 +54,7 @@ static void identityFinalizer(SEXP identity_) {
     }
 }
 
-Identity* authenticateWithId(SEXP con_, SEXP uuid_, SEXP ip_address_) {
+Identity* authenticateWithId(SEXP con_, SEXP uuid_, SEXP ip_address_, SEXP isEmrsId_) {
```

Comment:
Same nit re camelCase when the others aren't.

Reply:
in this case, I can go with snake_case since that's consistent with the function - is_emrs_id?

Reply:
Sounds good.